src/System.Private.CoreLib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -249,7 +249,10 @@ internal static unsafe int GetEnvironmentVariable(string lpName, Span<char> lpVa
         [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Unicode)]
         internal static extern unsafe bool FreeEnvironmentStrings(char* pStrings);
 
-        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Auto, SetLastError = true)]
+        [DllImport(Interop.Libraries.Kernel32, CharSet = CharSet.Auto)]
+        internal static extern int GetCurrentThreadId();
```

Comment:
We currently have:
https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/kernel32/Interop.GetCurrentThreadId.cs
Should we create this in CoreLib's interop folder similarly and then when it gets mirrored share it from corefx?

Reply:
This is supporting EventSource implementation. The EventSource has its own setup that it does not follow the coding conventions. I believe it was done to allow easier sharing of the code with .NET Framework originally, but I am not sure whether it is still relevant.

@brianrob @vancem Do you an opinion on whether it would make sense to cleanup EventSource to follow the general CoreCLR/CoreFX coding conventions?

Reply:
This was done when we were copying source back and forth on the desktop.  It's probably OK to clean this up, but @noahfalk and @jorive are the new owners, so they should confirm.